build-tools/pnpm-lock.yaml

```diff
@@ -2432,6 +2432,16 @@ packages:
       minimatch: 5.1.6
       mkdirp: 1.0.4
       path-browserify: 1.0.1
+    dev: true
+
+  /@ts-morph/common/0.22.0:
+    resolution: {integrity: sha512-HqNBuV/oIlMKdkLshXd1zKBqNQCsuPEsgQOkfFQ/eUKjRlwndXW1AjN9LVkBEIukm00gGXSRmfkl0Wv5VXLnlw==}
+    dependencies:
+      fast-glob: 3.3.2
+      minimatch: 9.0.3
+      mkdirp: 3.0.1
+      path-browserify: 1.0.1
+    dev: false
 
```

Comment:
What pnpm version are you using? I've noticed a lot of explicit dev: false showing up in your changes. I like explicit things but this is not consistent with the rest of the file and I have concern that when others touch the lock file these will go away causing undue noise and maybe road bumps.

Reply:
Everyone on the team should be using the same version (7.33.6) because we use [corepack](https://nodejs.org/docs/latest-v18.x/api/corepack.html).

Reply:
Then perhaps something in your workflow. I love pnpm and would love it more without some of its quirks.

Reply:
I tend to bounce between branches a lot... might be part of the cause.

Reply:
branch changes shouldn't do that